src/components/Sidebar.jsx

```diff
@@ -0,0 +1,323 @@
+                <Box width="16px" />
+              </>
+            )}
+          </Item>
+          {hasChildren && (
+            <ChildrenContainer
+              id={`sidebar-children-${id}`}
+              deployed={deployedIds.includes(id)}
+              height={`${deployedIds.includes(id) ? childrenHeights[id] || 0 : 0}px`}
+              overflow="hidden"
+              flex={{ shrink: 0 }}
+            >
+              <div>
+                {renderItems(items, marginLeft + 6)}
+              </div>
+            </ChildrenContainer>
+          )}
+        </Fragment>
+      )
+    })
+  }
+
+  return (
+    <Container
+      width={`${collapsed ? 74 : 208}px`}
+      background="background-front"
+      border={{
+        color: 'border',
+        size: 'small',
+        side: 'right',
+      }}
+      pad="24px 0px 16px 16px"
+      flex={{ grow: 0, shrink: 0 }}
+      overflow="scroll"
+    >
+      <Box
+        overflow="scroll"
+        flex="grow"
+        style={{ maxHeight: sidebarContentMaxHeight, height: sidebarContentMaxHeight }}
+      >
+        <Box
+          id="sidebar-items"
+          pad={{ right: '16px' }}
+        >
+          {renderItems(items)}
+        </Box>
+      </Box>
+      <Box
+        ref={sidebarBottomRef}
+        flex={{ grow: 0, shrink: 0 }}
+      >
+        <Box
+          direction="row"
+          align="center"
+          overflow="hidden"
+          margin={{ top: '16px' }}
+        >
+          <CollapseIconContainer
+            collapsed={collapsed}
+            background="background-light"
+            round="full"
+            align="center"
+            justify="center"
+            width="24px"
+            height="24px"
+            onClick={toggleCollapsed}
+          >
+            <CollapseIcon
+              color="text-xweak"
+              size={6}
+            />
+          </CollapseIconContainer>
+          <TransitionTextNoSelect
+            collapsed={collapsed}
+            size="small"
+            color="text-xweak"
+            margin={{ left: '16px' }}
+          >
+            Collapse
+          </TransitionTextNoSelect>
+        </Box>
+        <Box
+          direction="row"
+          align="center"
+          margin={{ top: '16px' }}
+        >
+          <Avatar
+            name={user.name}
+            imageUrl={user.imageUrl}
+          />
+          <Box pad={{ left: '8px' }}>
+            <TransitionText
+              collapsed={collapsed}
+              truncate
+              weight="bold"
+              size="small"
+              color="text-xweak"
+            >
+              {user.name}
+            </TransitionText>
+            <TransitionText
+              collapsed={collapsed}
+              truncate
+              size="xsmall"
+              color="text-xweak"
+            >
+              {user.email}
+            </TransitionText>
+          </Box>
+        </Box>
+      </Box>
+    </Container>
+  )
+}
+
+export default Sidebar
```

Comment:
Just realized this, but should we be using default exports?  Feel like that's going to make it more tricky to export everythign at the node module level

Reply:
in index.js:  `export { default as Button } from './components/Button'`, and then anywhere:
`import { Button } from 'plural-design-system'`

Reply:
yeah although if they were just standard exports, can't you do `export * from './components/Button'` which is a bit cleaner?  I seem to remember this is recommended for that reason

Reply:
If you agree with [this](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad) we may use named exports. I still prefer default but it's your call now :)